Source/DataSources/PointVisualizer.js

```diff
@@ -90,82 +90,43 @@ define([
-
-            var colorProperty = pointGraphics._color;
-            var outlineColorProperty = pointGraphics._outlineColor;
-
-            var newColor = Property.getValueOrDefault(colorProperty, time, defaultColor, color);
-            var newOutlineColor = Property.getValueOrDefault(outlineColorProperty, time, defaultOutlineColor, outlineColor);
-            var newOutlineWidth = Math.round(Property.getValueOrDefault(pointGraphics._outlineWidth, time, defaultOutlineWidth));
-            var newPixelSize = Math.max(1, Math.round(Property.getValueOrDefault(pointGraphics._pixelSize, time, defaultPixelSize)));
-
-            if (newOutlineWidth > 0) {
-                billboard.scale = 1.0;
-                needRedraw = needRedraw || //
-                             newOutlineWidth !== item.outlineWidth || //
-                             newPixelSize !== item.pixelSize || //
-                             !Color.equals(newColor, item.color) || //
-                             !Color.equals(newOutlineColor, item.outlineColor);
-            } else {
-                billboard.scale = newPixelSize / 50.0;
-                newPixelSize = 50.0;
-                needRedraw = needRedraw || //
-                             newOutlineWidth !== item.outlineWidth || //
-                             !Color.equals(newColor, item.color) || //
-                             !Color.equals(newOutlineColor, item.outlineColor);
-            }
-
-            if (needRedraw) {
-                item.color = Color.clone(newColor, item.color);
-                item.outlineColor = Color.clone(newOutlineColor, item.outlineColor);
-                item.pixelSize = newPixelSize;
-                item.outlineWidth = newOutlineWidth;
-
-                var centerAlpha = newColor.alpha;
-                var cssColor = newColor.toCssColorString();
-                var cssOutlineColor = newOutlineColor.toCssColorString();
-                var textureId = JSON.stringify([cssColor, newPixelSize, cssOutlineColor, newOutlineWidth]);
-
-                billboard.setImage(textureId, createCallback(centerAlpha, cssColor, cssOutlineColor, newOutlineWidth, newPixelSize));
-            }
+            pointPrimitive.show = true;
+            pointPrimitive.position = position;
+            pointPrimitive.scaleByDistance = Property.getValueOrUndefined(pointGraphics._scaleByDistance, time, scaleByDistance);
+            pointPrimitive.color = Property.getValueOrDefault(pointGraphics._color, time, defaultColor, color);
+            pointPrimitive.outlineColor = Property.getValueOrDefault(pointGraphics._outlineColor, time, defaultOutlineColor, outlineColor);
+            pointPrimitive.outlineWidth = Property.getValueOrDefault(pointGraphics._outlineWidth, time, defaultOutlineWidth);
+            pointPrimitive.pixelSize = Property.getValueOrDefault(pointGraphics._pixelSize, time, defaultPixelSize);
```

Comment:
We probably need to add a `scene.clampPointSize` similar to `scene.clampLineWidth` to avoid assigning a size the card/drivers don't support.  I also assume all cards have fairly large maximums?